akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -348,7 +348,8 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
         .info("Starting")
         // not counting for example "akkaSource", but it shouldn't have any other entries
         .withCustom(logEvent =>
-          logEvent.mdc.keysIterator.forall(entry => entry.startsWith("akka") || entry == "sourceActorSystem"))
+          logEvent.mdc.keysIterator.forall(entry =>
+            entry.startsWith("akka") || entry == "sourceActorSystem" || entry == "static"))
```

Comment:
this doesn't test that `"static" -> "1"` is included in MDC of the "starting" logging

Reply:
Done!